buildSrc/src/main/kotlin/kotlin-jvm.gradle.kts

```diff
@@ -1,29 +0,0 @@
-// The code in this file is a convention plugin - a Gradle mechanism for sharing reusable build logic.
```

Comment:
removed this because the project is pretty small right now and it is easier to just get rid of it rather than fight with it about plugin versions